Calibration/HcalAlCaRecoProducers/plugins/AlCaHBHEMuonFilter.cc

```diff
@@ -33,7 +33,7 @@
 #include "Geometry/CaloGeometry/interface/CaloGeometry.h"
 #include "Geometry/Records/interface/CaloGeometryRecord.h"
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```

Comment:
Shouldn't these debug settings get reverted (perhps remnant of some debug at the development time)?